views/includes/trustee-macros.html

```diff
@@ -1,13 +1,13 @@
 {% macro readableTrusteeRole(role) %}
-  {% if role === 'Beneficial_Owner' %}
+  {% if role === 'BENEFICIAL_OWNER' %}
```

Comment:
So this addressing issue 2 in in the defect. I think PR description now needs updating.